e2e/testconfig/testconfig.go

```diff
@@ -114,8 +114,30 @@ func FromEnv() TestConfig {
 			Tag:    chainBTag,
 			Binary: chainBinary,
 		},
-		RlyTag:     rlyTag,
-		UpgradeTag: upgradeTag,
+		UpgradeTag:    upgradeTag,
+		RelayerConfig: GetRelayerConfigFromEnv(),
+	}
+}
+
+// GetRelayerConfigFromEnv returns the RelayerConfig from present environment variables.
+func GetRelayerConfigFromEnv() relayer.Config {
+	relayerType := strings.TrimSpace(os.Getenv(RelayerTypeEnv))
+	if relayerType == "" {
+		relayerType = defaultRelayerType
+	}
+
+	rlyTag := strings.TrimSpace(os.Getenv(RelayerTagEnv))
+	if rlyTag == "" {
+		if relayerType == relayer.Cosmos {
+			rlyTag = defaultCosmosRelayerTag
+		}
+		if relayerType == relayer.Hermes {
+			// TODO: set default hermes version
+		}
+	}
```

Comment:
this one env var has to have 2 different defaults depending on the relayer type. If additional relayers are added, this could be moved to a map, with just two I decided to keep it simple.